### PR TITLE
Jobs management

### PR DESCRIPTION
**Pull request by ciprianc**
Install jobs from a git repo

### CODE REVIEW

File: jenkins/map.jinja [hunk 1]
```diff
@@ -11,6 +11,7 @@
             'name': 'Your humble Jenkins',
         },
         'optional_groups': [],
+        'repo_location': "",
```
Comment (ashb) [inline jenkins/map.jinja]: Your SLS looks at `jenkins.jobs.repo_location` but you are creating `jenkins.repo_location`

**Comment by mattmb** [comment]
LGTM :+1:

**Comment by ashb** [comment]
How are the creds for checking this repo out managed, or are we planning on pointing at a public repo to clone?

**Comment by ciprianc** [comment]
closing pr to make it a bit pretty and clearer. Will reopen in a bit

**Pull request closed without merging**